visualLocTanslational.m

```diff
@@ -113,32 +117,64 @@
 
             % collect the responses and appends to the event structure for
             % saving in the tsv file
-            responseEvents = collectAndSaveResponses(cfg, logFile, cfg.experimentStart);
-
-            responseEvents = getResponse('check', cfg.keyboard.responseBox, cfg, getOnlyPress);
-
+            responseEvents = getResponse('check', cfg.keyboard.responseBox, cfg, ...
```

Comment:
This part and and the one line 154 need refactoring but the logic is to check responses from KbQueue and then save them and change the name of the trial type from `response` to `trigger` if necessary